runtime/crab/src/pallets/assets.rs

```diff
@@ -24,16 +24,21 @@ pub enum AssetIds {
 	CKton = 1026,
 }
 
+frame_support::parameter_types! {
+	pub Signers: Vec<AccountId> = vec![ROOT];
+}
+
 impl pallet_assets::Config for Runtime {
 	type ApprovalDeposit = ConstU128<0>;
 	type AssetAccountDeposit = ConstU128<0>;
 	type AssetDeposit = ConstU128<0>;
 	type AssetId = AssetId;
 	type Balance = Balance;
+	type CreateOrigin = frame_support::traits::AsEnsureOriginWithArg<
```

Comment:
Could you please use `frame_support::traits::{AsEnsureOriginWithArg, EnsureSignedBy, IsInVec}`? This makes the code look much tighter.